src/components/generic/MasterContainer.tsx

```diff
@@ -21,8 +21,6 @@ const styles = StyleSheet.create({
   },
   innerMasterContainer: {
     padding: 20,
-    paddingBottom: 0,
```

Comment:
Deze twee regels zou ik eigenlijk weer terug zetten, nu krijg je op IOS weer zo'n wit balkje bij t scrollen
![Schermafbeelding 2022-04-13 om 09 47 11](https://user-images.githubusercontent.com/33700526/163126365-7c2133dd-a141-44e6-ba8c-7eea1b777725.png)

Reply:
Ja op android krijg je ook die balkjes, maar nu heb je wel die padding tussen de header en de content. Je kan ook marginBottom toevoegen aan de header, maar dan krijg je hetzelfde witte balkje.

Reply:
Hmm dat is wel minder eigenlijk. Weet niet of we die padding willen? Vond het zonder er ook gewoon goed uitzien op IOS eigenlijk

Reply:
![image](https://user-images.githubusercontent.com/45880258/163132739-3e87995e-496d-45cd-bc98-619524e02f09.png)
Dit was de oplossing van milo toch?

Reply:
Met die oplossing heb je dezelfde witte balk. Je kan het alleen weghalen eigenlijk, zorgt er wel voor dat de title tegen de header staat. Nu kunnen we met de stylesheet components straks gewoon een paddding aan de titel meegeven wat dit fixt.

Reply:
Lijkt niet uit te maken, krijg je alsnog die balk alleen dan kleiner
![Schermafbeelding 2022-04-13 om 10 47 31](https://user-images.githubusercontent.com/33700526/163137778-8a959c10-9d07-476f-bcf2-4c4101aab776.png)